students/MikeShand/lesson04/dict_lab.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+dictionary = {"name": "Chris", "city": "Seattle", "cake": "chocolate"}
+
+print(dictionary)
+
+dictionary.popitem()
+
+print(dictionary)
+
+dictionary["fruit"] = "Mango"
+
+print(dictionary.keys())
+
+print(dictionary.values())
+
+print("cake" in dictionary.keys())

```

Comment:
note that you can also do `"cake" in dictionary` without calling `keys()` as it will iterate over keys by default